CHANGELOG.txt

```diff
@@ -1,3 +1,16 @@
+2.0.0
+===================================
```

Comment:
we put `master` here for the ongoing changes in between releases, but we can change that before merging, no problem (also underline should match length of "heading")

Reply:
will change it (yéééé for the ==)